src/ngg/runtime_fns.jl

```diff
@@ -14,6 +14,8 @@ Base.show(io::IO, rtfn::RuntimeFn{Args, Kwargs, Body, Name}) where {Args, Kwargs
         repr = "$Name = ($args;$kwargs) -> $body"
         print(io, repr)
 end
```

Comment:
This will disable the chance of viewing the generated expressions conveniently.
How about adding some mechanism to automatic numbering?
This will allow user to view the expressions via

```julia
details(114514)
```
where `114514` is the number of some generated function